tar_scm.py

```diff
@@ -761,6 +761,52 @@ def detect_changes_commands_git(repodir, subdir, changes):
     return changes
 
 
+def detect_changes_commands_svn(repodir, changes):
+    """Detect changes between GIT revisions."""
+    last_rev = changes['revision']
+    new_lines = []
+
+    if last_rev is None:
+        last_rev = safe_run(['svn', 'log', '-l10', '-q', '--incremental'], cwd=repodir)[1].split('\n')
```

Comment:
This `if` clause could be in a separate function, since this one is currently too long.